src/backend/distributed/commands/utility_hook.c

```diff
@@ -731,12 +732,29 @@ ExecuteDistributedDDLJob(DDLJob *ddlJob)
 		}
 		PG_CATCH();
 		{
-			ereport(ERROR,
-					(errmsg("CONCURRENTLY-enabled index command failed"),
-					 errdetail("CONCURRENTLY-enabled index commands can fail partially, "
-							   "leaving behind an INVALID index."),
-					 errhint("Use DROP INDEX CONCURRENTLY IF EXISTS to remove the "
-							 "invalid index, then retry the original command.")));
+			MemoryContextSwitchTo(savedContext);
+			ErrorData *edata = CopyErrorData();
+
+			/*in concurrent index creation */
+			/*if a worker index with the same name already exists, prompt to DROP the current index and retry original command */
+			if (edata->sqlerrcode == ERRCODE_DUPLICATE_TABLE)
+			{
+				ereport(ERROR,
+						(errmsg("CONCURRENTLY-enabled index command failed"),
```

Comment:
is there a test that goes into this branch of if?

Reply:
Yes it was already there.
https://github.com/citusdata/citus/blob/master/src/test/regress/sql/multi_index_statements.sql#L178